devicetypes/smartthings/dawon-zwave-smart-switch.src/dawon-zwave-wall-smart-switch.groovy

```diff
@@ -0,0 +1,392 @@
+/**
+ *  Copyright 2018 SmartThings
```

Comment:
nit: copyright date

Reply:
Thanks. I will update to "Copyright 2020 SmartThings".